src/main/resources/application.properties

```diff
@@ -1 +1,6 @@
+spring.datasource.url=jdbc:postgresql://localhost:5432/employee
+spring.datasource.username=astro
+spring.datasource.password=astro063
 
+spring.jpa.hibernate.naming.implicit-strategy=org.hibernate.boot.model.naming.ImplicitNamingStrategyLegacyJpaImpl
```

Comment:
Any reason to explicitly specifying these naming strategies?

Reply:
I have already created tables using camel cases.
so to override the implicit implementation of jpa naming convention.